### PR TITLE
[extension] use tailwincss in popup for styling

### PR DESCRIPTION
**Pull request by flocks**
:warning: I need to rebase on latest main
:warning: Not really ready pretty sure it breaks the style of other pages, but for some reason I don't manage to trigger them (click on `rate note` on a note doesn't seem to work anymore for some reason, need to investigate)

I started to include tailwind, especially on the `popup` page. I have also move some code to separate files to increase code readability. 
I've also updated webpack config so we can use path alias `@` that refers to `src/` folder. So we don't have to do relative imports. 

That's only a first pass, it needs some polishing
demo

### CODE REVIEW

**Comment by YBadiss** [comment]
btw @flocks

> click on rate note on a note doesn't seem to work anymore for some reason, need to investigate

that was caused by a bug in the backend url, fixed since then on main 👍

File: fc-community-extension/src/utils/backend.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-const BACKEND_URL = 'https://api.factchain.tech';
+// const BACKEND_URL = 'https://api.factchain.tech';
+const BACKEND_URL = 'https://fc-community-backend-15f6c753d352.herokuapp.com';
```
Comment (flocks) [inline fc-community-extension/src/utils/backend.js:2]: revert

**Comment by flocks** [comment]
> that was caused by a bug in the backend url, fixed since then on main 👍

AH thanks, will retry so we can move forward

**Comment by j68847945** [comment]
done in https://github.com/factchain/factchain-community/pull/79

**Pull request closed without merging**